dbt_coverage/__init__.py

```diff
@@ -337,7 +354,14 @@ def find_new_misses(self):
 
         new_misses_names = self.after.misses - (self.before.misses if self.before is not None
                                                 else set())
-        new_misses_entity_names = set(miss.split('.', maxsplit=1)[0] for miss in new_misses_names)
+
+        # Table names are in the form schema_name.table_name
```

Comment:
@sweco since there are all these nice objects set up in here, perhaps each of the entities could have direct access to the name of the table and wouldn't need to split it this way?

Please only take this as a suggestion -- I am sure you've considered something like this and it turned out the splitting approach was in fact easier / better fit in some sense 🙂

Reply:
This is indeed not very nice, thanks for motivating me into refactoring this. 😊  Maybe I should create an object representing a `Miss` to get rid of this mess. If it was created as follows, we would get rid of the parsing of the strings that we concatenated before.

```python
@dataclass
class Miss:
    table: str
    column: str
```

The problem with what you suggested is that the objects you're mentioning (`Catalog`, `Table` and `Column`) are only used when computing coverage and not when comparing it and here we're comparing so we don't have them.

Reply:
@mrshu, I finally fixed this by using a new class `ColumnRef`. I believe that it's much cleaner now without the string parsing. 😊 After all, one should use domain objects instead of doing some magic with strings almost all the time.